.flake8

```diff
@@ -7,3 +7,4 @@ exclude = ui/,build*,definitions
 per-file-ignores =
     nexus_constructor/geometry/__init__.py:F401
     tests/ui_tests/test_ui_fields.py:F811,F401
+    tests/*:E402,F821
```

Comment:
What are these for? Are you allowed comments in this config file? Would be nice to record why these codes are ignored.

Reply:
Have added a comment to explain in the flake8 file, did try it previously but the comment caused issues. seems like you have to put the comment on a separate line in the .flake8 file